core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,1937 @@
-      "parameters": [
-        {
-          "$ref": "#/components/parameters/groupid"
-        }
-      ],
-      "get": {
-        "operationId": "get{%-resourceNamePlural-%}All",
-        "description": "Get an optionally filtered collection of resources",
-        "parameters": [
-          {
-            "in": "query",
-            "name": "inline",
-            "description": "Set if references shall be inlined",
-            "required": false,
-            "allowEmptyValue": true,
-            "schema": {
-              "type": "boolean"
-            }
-          },
-          {
-            "in": "query",
-            "name": "skip",
-            "description": "The number of resources to skip",
-            "required": false,
-            "schema": {
-              "type": "integer"
+    "components": {
+        "securitySchemes": {
+            "api_key": {
+                "type": "apiKey",
+                "name": "code",
+                "in": "query"
             }
-          },
-          {
-            "in": "query",
-            "name": "top",
-            "description": "The number of resources to show",
-            "required": false,
+        },
+        "parameters": {
+            "groupid": {
+                "name": "groupid",
+                "in": "path",
+                "required": true,
                 "schema": {
-              "type": "integer"
-        }
-      },
-          {
-            "in": "query",
-            "name": "filter",
-            "description": "Filter criterion(s)",
-            "explode": true,
-            "required": false,
-              "schema": {
-              "type": "array",
-              "items": {
-                "type": "string"
-                }
-              }
-            }
-        ],
-        "responses": {
-          "200": {
-            "description": "Updated list of resources",
-            "content": {
-              "application/json": {
+                    "type": "string"
+                },
+                "description": "The unique identifier of the group."
+            },
+            "resourceid": {
+                "name": "resourceid",
+                "in": "path",
+                "required": true,
                 "schema": {
-                  "type": "object",
-                  "additionalProperties": {
-                     "$ref": "{%-resourceTypeReference-%}"
-                  }
-                }
-              }
+                    "type": "string"
+                },
+                "description": "The unique identifier of the resource."
+            },
+            "versionid": {
+                "name": "versionid",
+                "in": "path",
+                "required": true,
+                "schema": {
+                    "type": "string"
+                },
+                "description": "The unique identifier of the resource version."
+            },
+            "resource-description": {
```

Comment:
xRegsitry-xxx

Reply:
wait... where does this appear? I see them as "headers" too, but where are these used?